config/passport.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
might be good, but I'd keep it separate to better portray the purpose and scope of the PR.
it works a charm both for the PR's author and all reviewers, apart from minimizing the risks of breaking completely unrelated things

Reply:
Yes, I can confirm the API that could work on postman quickly before starting to coding unit test. Now I don't think this is necessary